lib/app_web/controllers/auth_controller.ex

```diff
@@ -1,9 +1,38 @@
-defmodule AppWeb.Auth do
+defmodule AppWeb.AuthController do
   use AppWeb, :controller
-  # use AppWeb, :router
-  # import Plug.Conn
+  import Phoenix.LiveView, only: [assign_new: 3]
 
-  # def logout(conn) do
-  #   configure_session(conn, drop: true)
-  # end
+  def on_mount(:default, _params, %{"jwt" => jwt} = _session, socket) do
+
+    claims = jwt
+    |> AuthPlug.Token.verify_jwt!()
+    |> AuthPlug.Helpers.strip_struct_metadata()
+    |> Useful.atomize_map_keys()
+
+    IO.inspect(claims, label: "claims")
+    App.Person.upsert(claims)
```

Comment:
The only way to create a `person` in the MVP is via this `upsert/1` invocation. #115

Reply:
We've agreed to completely remove the `people` table from the project #118 ✂️